src/App.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import UserAuth from './Components/UserAuth';
+import UserSignUp from './Components/UserSignUp';
+import Layout from './Components/Layout';
+import PlaceList from './Components/PlaceList';
+import MyReservations from './Components/MyReservations';
+import './App.css';
+import DetailsPage from './Components/DetailsPage';
+import AddPlace from './Components/AddPlace';
+import AddReservation from './Components/AddReservation';
+import DeletePlace from './Components/DeletePlace';
+
+function App() {
```

Comment:
- Please, consider using arrow funcs according to Javascript ES6 good practices.